config/config.go

```diff
@@ -0,0 +1,181 @@
+package config
+
+import (
+	"errors"
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"strconv"
+	"strings"
+	"time"
+	"unicode"
+)
+
+type Application struct {
+	FilePath,
+	Postcode,
+	StartTimeString,
+	EndTimeString string
+	KeywordsString []string
+	StartTime,
+	EndTime int
+}
+
+func Init() *Application {
+	var config Application
+
+	var keywordsString, postcode, timeWindow, path string
+	flag.StringVar(&keywordsString, "keywords", "Potato,Veggie,Mushroom", "Comma-separated list of keywords to search for in recipe names")
+	flag.StringVar(&postcode, "postcode", "10120", "Postcode to search for")
+	flag.StringVar(&timeWindow, "time", "10AM-3PM", "Time window to search for in format 'start-end' (e.g., '10AM-3PM')")
+	flag.StringVar(&path, "file", "deliveries.json", "Path to file location")
+	flag.Parse()
+
+	err := validatePostcode(postcode)
+	if err != nil {
+		log.Println("Error:", err)
+		return nil
+	}
```

Comment:
для вывода в stdErr можно использовать такую реализацию
```suggestion

	errorLog := log.New(os.Stderr, "Config Error: ", 1)
	err := validatePostcode(postcode)
	if err != nil {
		errorLog.Println(err)
		return nil
	}
```

Reply:
Но выглядит как проще в этом методе также вернуть ошибку и логгировать её уже выше

Reply:
Исправил метод